.github/workflows/build-assets.yml

```diff
@@ -330,8 +330,8 @@ jobs:
           build-args: |
             PKG_FILE=${{ steps.ln-container.outputs.k8s_pkg_file }}
           tags: |
-            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.internal_tag }}
-            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.candidate_base_branch }}
+            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/hpccpl-docker-local/hpcc-platform/${{ matrix.os }}/platform-core-ln:${{ needs.preamble.outputs.internal_tag }} 
```

Comment:
"hpccpl-docker-local" probably ok , but suspect it should be configurable via e.g. JFROG_REGISTRY_PATH or similar.

Reply:
trivial: an unnecessary new blank space has been added to the end of the line.

Reply:
Good point. That will enable custom builds as well and save me from an if statement. I'll make the change.